tempodb/encoding/versioned.go

```diff
@@ -78,7 +81,7 @@ func DefaultEncoding() VersionedEncoding {
 
 // LatestEncoding returns the most recent encoding.
 func LatestEncoding() VersionedEncoding {
-	return vparquet3.Encoding{}
+	return vparquet4.Encoding{}
```

Comment:
i'm concerned about this. if we merge vp4 primarily to prevent it from rotting and then don't roll it out for awhile this change will prevent our ingester tests from running vp3. wdyt?

Reply:
after some thought, i'm good on being more aggressive here. let's keep this change to update `LatestEncoding()`